pages/dashboard/projects.vue

```diff
@@ -319,179 +319,146 @@ import SaveIcon from '~/assets/images/utils/save.svg'
 import AscendingIcon from '~/assets/images/utils/sort-asc.svg'
 import DescendingIcon from '~/assets/images/utils/sort-desc.svg'
 
-export default defineNuxtComponent({
-  components: {
-    Avatar,
-    Badge,
-    SettingsIcon,
-    TrashIcon,
-    Checkbox,
-    IssuesIcon,
-    PlusIcon,
-    CrossIcon,
-    EditIcon,
-    SaveIcon,
-    Modal,
-    ModalCreation,
-    Multiselect,
-    CopyCode,
-    AscendingIcon,
-    DescendingIcon,
-  },
-  async setup() {
-    const { formatMessage } = useVIntl()
+const { formatMessage } = useVIntl()
+
+useHead({
+  title: 'Projects - Modrinth',
+})
 
-    const user = await useUser()
-    await initUserProjects()
-    return { formatMessage, user: ref(user) }
+const data = useNuxtApp()
+const user = await useUser()
+await initUserProjects()
+
+const editLinksModal = ref(null)
+
+const projects = ref(updateSort(user.value.projects, 'Name'))
+const selectedProjects = ref([])
+const sortBy = ref('Name')
+const descending = ref(false)
+const editLinks = ref({
```

Comment:
`editLinks` can be a `reactive` object instead.

Reply:
Fixed ^^